modules/op-release-notes-1-13.adoc

```diff
@@ -0,0 +1,141 @@
+
+* With this update, optionally, you can enable a setting so that OpenShift Pipelines stops the pod for the task run but does not delete the pod. To enable this setting, set the `pipeline.options.configMaps.feature-flags.data.keep-pod-on-cancel` spec to `true` and the `pipeline.options.configMaps.feature-flags.data.keep-pod-on-cancel` spec to `alpha` in the `TektonConfig` custom resource (CR).
+
+* Before this update, you had to enable alpha features in order to set compute resource limits on the task level. With this update, you can use the `computeResources` spec for a TaskRun CR to set the resource limits for a task.
+
+* With this update, when specifying a task and using the `displayName` parameter, you can use parameters that include the values of parameters, results, or context variables in the display name, for example, `$(params.application)`, `$(tasks.scan.results.report)`, `$(context.pipeline.name)`.
+
+* With this update, when specifying a remote pipeline or task using the hub resolver, in the `version` parameter you can use inequation constraints such as `>=0.2.0,< 1.0.0`.
+
+* With this update, when specifying a task you can use a Common Expression Language (CEL) expression in the `when` expression. To use this feature, you must set the `pipeline.options.configMaps.feature-flags.data.enable-cel-in-whenexpression` spec to `true` in the `TektonConfig` CR.
+
+* With this update, when specifying a pipeline in a `PipelineRun` CR spec, you can reference the results produced by an inline task in a subsequent inline task.
++
+.Example usage
+[source,yaml]
+----
+  apiVersion: tekton.dev/v1
+  kind: Task
+  metadata:
+    name: uid-task
+  spec:
+    results:
+      - name: uid
+    steps:
+      - name: uid
+        image: alpine
+        command: ["/bin/sh", "-c"]
+        args:
+          - echo "1001" | tee $(results.uid.path)
+  ---
+  apiVersion: tekton.dev/v1
+  kind: PipelineRun
+  metadata:
+    name: uid-pipeline-run
+  spec:
+    pipelineSpec:
+      tasks:
+      - name: add-uid
+        taskRef:
+          name: uid-task
+      - name: show-uid
+        taskSpec:
+          steps:
+            - name: show-uid
+              image: alpine
+              command: ["/bin/sh", "-c"]
+              args:
+                - echo $(tasks.add-uid.results.uid)
+----
+
+* With this update, when configuring the cluster resolver, you can set the value of the `blocked-namespaces` parameter to `*`. With this setting, only the namespaces listed in the `allowed-namespaces` parameter are allowed and all other namespaces are blocked.
+
+[id="operator-new-features-1-13_{context}"]
+=== Operator
+
+* With this update, the `disable-affinity-assistant` feature flag has been deprecated and will be removed in a future release. Instead, in the `TektonConfig` CR, you can set the `pipeline.options.configMaps.feature-flags.data.keep-pod-on-cancel` spec to one of the following values:
```

Comment:
It's `pipeline.options.configMaps.feature-flags.data.coschedule`

Reply:
fixed